burrito-fg/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "burrito-fg"
+version = "0.1.0"
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+gdnative = "0.9"
+breadx = {git="https://github.com/royalmustard/breadx"}
```

Comment:
Is there a reason we are not using the mainline breadx crate?

Reply:
Mainline breadx has a bug causing it to panic when it deserializes an empty string property from X. I already mad a PR, but for now we are stuck with the fix

Reply:
SGTM, can you add a new issue once this is merged to update this dependency whenever your pr gets merged into breadx?